docs/general/adapters.md

```diff
@@ -194,6 +194,43 @@ The user could pass in `include=**`.
 
 We recommend filtering any user-supplied includes appropriately.
 
+#### Include Parameters
+
+You may wish to include related resources based on the `include` request
+parameter (See [fetching_includes](http://jsonapi.org/format/#fetching-includes). You can opt-in to this pattern:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  associations_via_include_param(true)
```

Comment:
This PR has always made me a little nervous so I've mostly avoided reading it and relying on others.. but now I'm a little more concerned about some of the design choices, so will take a look.  It's hard to balance trying to do what's right for the code base and wanting to use the solutions and work-product in pull requests.  So, my interest right now is to make this PR is good for AMS, not just a good feature or good code.. that make sense?